public/index.html

```diff
@@ -10,10 +10,10 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <script src='https://cdn.jsdelivr.net/npm/zone.js@0.11.4/bundles/zone.umd.min.js'></script>
-    <script src="https://cdn.jsdelivr.net/npm/@laserfiche/lf-ui-components@13.0/cdn/lf-ui-components.js" defer></script>
-    <link href="https://cdn.jsdelivr.net/npm/@laserfiche/lf-ui-components@13.0/cdn/lf-laserfiche-lite.css" rel="stylesheet" />
-    <link href="https://cdn.jsdelivr.net/npm/@laserfiche/lf-ui-components@13.0/cdn/indigo-pink.css" rel="stylesheet" />
+    <script src="https://lfxstatic.com/npm/@laserfiche/lf-ui-components@13/cdn/lf-ui-components.js" defer></script>
+    <script src='https://lfxstatic.com/npm/-/zone.js@0.11.4/bundles/zone.umd.min.js'></script>
+    <link href="https://lfxstatic.com/npm/@laserfiche/lf-ui-components@13/cdn/lf-laserfiche-lite.css" rel="stylesheet" />
```

Comment:
doesn't zone have to be loaded before the ui-components script?

Reply:
I think since zone is way smaller than ui-components, and ui-components has the defer tag, zone can be loaded later but will still finish loading before ui-components

Reply:
I think you're right, but let's put it first since that's what we document and it makes it easier to understand